src/common/grpc/src/channel_manager.rs

```diff
@@ -381,6 +385,16 @@ impl ChannelConfig {
         self.client_tls = Some(client_tls_option);
         self
     }
+
+    /// Enable compression for gRPC
+    ///
+    /// Disabled by default.
+    pub fn accept_compressed(self, compression: Vec<CompressionEncoding>) -> Self {
```

Comment:
I don't find the logic a config option can manipulate this option.

Also, is it possible that we enable compression by default? It seems there is no downside for doing so.

Reply:
It's confirmed that `channel_manager` and `client` is only used internally, we can remove those configs now. Should we also enable gzip/zstd compression encoding by default internally(between frontend/datanode/metasrv)? @sunng87 @waynexia @fengjiachun 

> Also, is it possible that we enable compression by default? It seems there is no downside for doing so.

Now the compression support is auto-enabled on our server(namely frontend), but it depends on the client request to determine whether to use compression or not(the compression encoding is done on client side).

Reply:
> Now the compression support is auto-enabled on our server(namely frontend), but it depends on the client request to determine whether to use compression or not(the compression encoding is done on client side).

Of course. We can "allow" the support. Whether or not it's compressed is based on the request - this is normal :D